lib/config/config-file.js

```diff
@@ -348,31 +384,39 @@ function applyExtends(config, filePath) {
  */
 function resolve(filePath) {
 
-    if (isFilePath(filePath)) {
-        return path.resolve(filePath);
-    } else {
-
-        // it's a package
-
-        if (filePath.charAt(0) === "@") {
-            // it's a scoped package
-
+    /**
+     * Brings package name to correct format based on prefix
+     * @param {string} name The name of the package.
+     * @param {string} prefix Can be either "eslint-plugin" or "eslint-config
+     * @returns {string} Normalized name of the package
+     * @private
+     */
+    function normalizePackageName(name, prefix) {
+        if (name.charAt(0) === "@") {
+        // it's a scoped package
```

Comment:
Tiny nit: This line lost an indent level during changes.

I wouldn't have said anything at all, but I think the comments in here are now slightly out of date.